raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -349,12 +352,16 @@ def handle_onchain_secretreveal(
     the current lock removed from the merkle tree and the transferred amount
     updated.
     """
-    is_valid_secret = state_change.secrethash == initiator_state.transfer.lock.secrethash
+    valid_secret = is_valid_secret_reveal(
```

Comment:
The below checks are called `is_X_Y_Z`... this is breaking the convention

Reply:
Hmmm I don't like the convention. Can change it all if you want. `is_X_Y_Z` in my mind should be a function call.

Reply:
Anyway I shouldn't break this in this PR anyhow. Will rename to `is_valid_secret` for now